parser/balance_changes.go

```diff
@@ -128,9 +128,11 @@ func (p *Parser) BalanceChanges(
 		}
 	}
 
-	allChanges := []*BalanceChange{}
+	i := 0
```

Comment:
Ahh I'm not a go expert but I should have seen this that the key is string. ACK this thanks!